tests/bin/action_runner_test.py

```diff
@@ -51,40 +40,56 @@ class RegisterTestCase(TestCase):
     def patch_sys(self):
         with contextlib.nested(
             mock.patch('action_runner.os.path.isdir', autospec=True),
-            mock.patch('action_runner.os.makedirs', autospec=True),
+            mock.patch('action_runner.os.access', autospec=True),
             mock.patch('action_runner.StatusFile', autospec=True),
         ) as (
             self.mock_isdir,
-            self.mock_makedirs,
```

Comment:
Might want to keep the mock of makedirs in this setup_teardown. That way, you don't have to remember to mock makedirs in every test that calls it, to avoid touching the filesystem.